philistine/mne/io.py

```diff
@@ -22,13 +21,14 @@
 # binary_int_16, ascii as future formats?
 
 
-supported_formats = {'binary_float32' : 'IEEE_FLOAT_32',
-                     'binary_int16'   : 'INT_16'}
+supported_formats = {'binary_float32': 'IEEE_FLOAT_32',
+                     'binary_int16': 'INT_16'}
 
 supported_orients = set(['multiplexed'])
 
+
 def write_raw_brainvision(raw, vhdr_fname, events=True):
-    """Write raw data to BrainVision format.
+    """Write raw data to Brainvision format.
```

Comment:
Capitalizing the "V" is again intentional, to match the way Brain Products stylizes their brand.